src/sentry/integrations/vsts/issues.py

```diff
@@ -28,7 +28,7 @@ def create_default_repo_choice(self, default_repo):
         project = self.get_client().get_project(self.instance, default_repo)
         return (project["id"], project["name"])
 
-    def get_project_choices(self, group, **kwargs):
+    def get_project_choices(self, group=None, **kwargs):
```

Comment:
@ceorourke Nit: I would add `project` as an optional arg here `project=None` so we don't need to do `kwargs.get("project")`